src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs

```diff
@@ -173,15 +215,16 @@ public async Task CloseOutputAsync_ClientInitiated_CanReceive_CanClose(Uri serve
             {
                 var cts = new CancellationTokenSource(TimeOutMilliseconds);
 
-                var closeStatus = WebSocketCloseStatus.InvalidPayloadData;
+                // See issue for Browser websocket differences https://github.com/dotnet/runtime/issues/45538
+                var closeStatus = PlatformDetection.IsBrowser ? WebSocketCloseStatus.NormalClosure : WebSocketCloseStatus.InvalidPayloadData;
                 string closeDescription = "CloseOutputAsync_Client_InvalidPayloadData";
 
                 await cws.SendAsync(WebSocketData.GetBufferFromText(message), WebSocketMessageType.Text, true, cts.Token);
                 // Need a short delay as per WebSocket rfc6455 section 5.5.1 there isn't a requirement to receive any
                 // data fragments after a close has been sent. The delay allows the received data fragment to be
                 // available before calling close. The WinRT MessageWebSocket implementation doesn't allow receiving
                 // after a call to Close.
-                await Task.Delay(100);
+                await Task.Delay(PlatformDetection.IsBrowser ? 500 : 100); // default timeout is not enough for browser sometimes so we will extend it
```

Comment:
could we just use 500 for all platforms? This is already OuterLoop test so it may not matter much.

Reply:
Addressed in recent commits